python/cairo-core/tests/src/cairo_core/test_maths.py

```diff
@@ -182,3 +182,19 @@ def test_felt252_to_bytes_be_should_panic_on_wrong_output_noncanonical(
     def test_felt252_bit_length(self, cairo_run, value):
         res = cairo_run("felt252_bit_length", value=value)
         assert res == value.bit_length()
+
+    @given(
+        value=st.integers(min_value=2, max_value=2**248 - 1),
+        len=st.integers(min_value=0, max_value=50),
+    )
+    def test_felt252_to_bits_rev(self, cairo_run, value, len):
+        # when len is 0, we expect an empty list, otherwise if len is too small,
+        # we need to increase it to the bit length of the value + 1
```

Comment:
why so? what if I want to get, let's say, the 20 first bits of a number `n`?

Reply:
I just reread the issue description and I'm not sure we should allow a `len` argument (since the last steps verifies the decomposition corresponds to the initial value)

Reply:
yes, we can use a system of mask